src/test/java/com/ericsson/eiffel/remrem/generate/cli/CliUnitTests.java

```diff
@@ -63,7 +65,7 @@ public void testHandleFileArgsFail() throws Exception {
         assertTrue(CLIOptions.getErrorCodes().contains(code));		
     }
 
-    @Test
+    /*@Test
```

Comment:
Is it possible to enable this test case or describe why it has been commented out?